src/authorizer.js

```diff
@@ -11,6 +11,7 @@ class Authorizer {
    * @param {String}   configuration.jwkKeyListUrl url used to retrieve the jwk public keys
    * @param {String}   configuration.authorizerContextResolver function to populate the authorizer context, by default it will only contain the JWT. function(identity, token) {}
    * @param {String}   configuration.usagePlan An AWS Api Gateway usage plan ID, this will create an api key and attach it to the returned policy document.
+   * @param {String}   configuration.jwtVerifyOptions parameters that are passed to the JWT verification options. Defaults to { algorithms: ['RS256'] }
```

Comment:
What would you think instead of "going all the way" and providing a jwt verifier, with this one set as the default, so that we can both provide further extensibility?

```js
configuration.jwtIdentityProvider = (token, key) => {
  if (!key) {
    return await jwtManager.decode(token, { complete: true });
  }
    return await jwtManager.verify(token, key, { algorithms: ['RS256'] });
  }
}
```

Reply:
Mh... I'm a bit unsure.

Technically, I fully agree. That said, this library is fairly opinionated and the authorizer is actually an AWS Lambda Authorizer.

Since my use case is to actually just add the `audience` to the check, I might actually go the other way and just provide the audience as a specific parameter. Otherwise, I'll have to pull in and configure the jwtManager in all my client code (ending up in me proposing an `authorizerwrapper.js` to solve for this case once.

So yes, puristically I agree, but practically I think most use cases benefit from simply having a ready implementation.

(I still can be convinced. What do you think of my thoughts?)

Reply:
I'm trying to figure out how many fields should be configurable, surely `audience`, but it seems also `issuer`.  Seems like almost all of them, and they are fairly generic things for the library that alternatives would also use.  It would be one thing to support different token formats and have a converter for each, but it doesn't seem to be remotely necessary.

From your suggestion, I agree.